src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/AbstractPlatformProxy.cs

```diff
@@ -20,6 +20,10 @@ namespace Microsoft.Identity.Client.PlatformsCommon.Shared
 {
     internal abstract class AbstractPlatformProxy : IPlatformProxy
     {
+        public const string WindowsOS = "Windows";

```

Comment:
Make these names a bit more specific/clear what they are for - that these are sent to STS. WindowsOsPlatformName or something, or put in a nested class? Or I guess property comment also works.